src/blot-formatters/actions/image-delete-action.ts

```diff
@@ -0,0 +1,81 @@
+import BlotFormatter, { Action } from "quill-blot-formatter";
+import { Quill } from "quill";
+
+export class ImageDeleteAction extends Action {

```

Comment:
Recommend adding some JSDoc comments to this class/public methods. If not here, please open up a new issue